### PR TITLE
MNT: Re-rendered with conda-build 3.19.2 and conda-smithy 3.7.1

### PR DESCRIPTION
**Pull request by hufngvuowng**

### CODE REVIEW

**Comment by hufngvuowng** [comment]
Hi @tacaswell! Could you review this PR before it can be merged?

**Review by mrakitin — CHANGES_REQUESTED** [review]
I think the URL in `meta.yaml` (`git_url: https://github.com/xpdAcq/xpdAn.git`) should be changed to the tagged .tar.gz file from https://github.com/xpdAcq/xpdAn/releases/tag/0.8.1.

Also, could you please add a summary entry in the `about:` section?

**Comment by hufngvuowng** [comment]
Hi @mrakitin! Thank you for your feedback. I changed the git_url and added the summary but the tests failed. Could you check if the url I added was correct?

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
  url: https://github.com/xpdAcq/xpdAn/releases/download/{{ version }}/xpdAn-{{ version }}.tar.gz
```

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
  summary: Python package that aids data analysis at the NSLS-II XPD Beamline.
```

**Review by mrakitin — COMMENTED** [review]
Thanks @tienhungvuong! I put a couple of suggestions to solve the issue.

We also parametrize the name of the package and provide its `sha256` hash. Do you mind to also update it? https://github.com/nsls-ii-forge/bluesky-feedstock/blob/master/recipe/meta.yaml is a good example how it should look. Thanks!

**Comment by hufngvuowng** [comment]
Thank you very much @mrakitin! Where can I find the sha256 hash of the package?

**Comment by mrakitin** [comment]
Here are some recommendations: https://github.com/conda-forge/staged-recipes/wiki/Frequently-asked-questions#2-how-do-i-populate-the-hash-field.

I normally download the archive which is going to be used by the conda-build, and run this:
```bash
$ openssl sha256 <file name>
```

**Comment by mrakitin** [comment]
More formal instructions are here: https://conda-forge.org/docs/maintainer/adding_pkgs.html#populating-the-hash-field.

I prefer to download the file first to make sure it's a proper archive.

**Comment by hufngvuowng** [comment]
Hi @mrakitin! I made the changes you suggested. Could you review this PR to see if it can be merged?

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]: This should go to the `source` group. The rest looks OK.

**Review by mrakitin — CHANGES_REQUESTED** [review]
Please fix one more issue indicated below.
@tacaswell, do you mind to re-review the updates in `meta.yaml` and merge (I'll try to keep away from my computer for some time)? Thanks!

**Review by mrakitin — APPROVED** [review]
Thanks, @tienhungvuong!

**Comment by mrakitin** [comment]
@tienhungvuong, this package is missing the `xpdtools` dependency in this `nsls2forge` channel, so it took the missing dep from `conda-forge` instead. That's the behavior we want to avoid. Could you please build the missing dependency? Also, for one of the dependencies, there was a hard pin of databroker 0.13. That should be relaxed as we are going to deploy databroker 1.0 soon.

**Comment by hufngvuowng** [comment]
@mrakitin Sorry for the late reply because I didn't get the notifications for some reasons. I'm working on building xpdtools now. Could you create a repo for `xpdtools-feedstocks` so I can push to? Thank you very much!

**Comment by mrakitin** [comment]
Done via https://github.com/nsls-ii-forge/xpdtools-feedstock/pull/1. Once merged, please go ahead and update to reflect the latest/proper dependencies.

**Pull request merged**
